src/components/sidebar/sidebar.component.tsx

```diff
@@ -62,7 +67,7 @@ export interface SidebarProps extends PaddingProps, TagProps {
     | "large"
     | "extra-large";
   /** an optional array of refs to containers whose content should also be reachable by tabbing from the sidebar */
-  focusableContainers?: React.MutableRefObject<HTMLElement>[];
+  focusableContainers?: CustomRefObject<HTMLElement | null>[];
```

Comment:
```suggestion
  focusableContainers?: CustomRefObject<HTMLElement>[];
```

the `null` option shouldn't be needed here as that's taken care of in `CustomRefObject`